microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -21,33 +21,29 @@ ms.date: 09/13/2023
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Edit files with Workspaces (Filesystem tab)
+# Edit and save files in a workspace
+<!-- https://developer.chrome.com/docs/devtools/workspaces -->
 
-Use the **Filesystem** tab in the **Sources** tool to define a workspace, to save DevTools changes in your source code files rather than only in a transient copy of the files that are returned by the web server.
+Use the **Workspace** tab in the **Sources** tool to define a workspace, to save DevTools changes in your source code files rather than only in a transient copy of the files that are returned by the web server.
```

Comment:
```suggestion
Use the **Workspace** tab in the **Sources** tool to display and edit the contents of a folder on your computer from within the **Sources** tool.  The changes that you make to the files in a workspace are saved to your computer.

The typical way to set up a workspace is to set it up to display the source files of your website's frontend code. This way, you use DevTools both to inspect your website, and to make changes to its source files, and the changes that you make are saved on your computer, rather than lost when you refresh the page.
```

Reply:
```
Use the **Workspace** tab in the **Sources** tool to display and edit the contents of a folder on your computer from within the **Sources** tool.  The changes that you make to the files in a workspace are saved to your computer.

The typical way to set up a workspace is to set it up to display the source files of your website's frontend code.  This way, you use DevTools both to inspect your website, and to make changes to its source files.  This way, the changes that you make are saved on your computer, rather than lost when you refresh the page.
```